packages/@orchidui-vue/src/Form/Input/OcInput.vue

```diff
@@ -131,8 +143,8 @@ const inputClasses = computed(() => [
 
           <input
             ref="inputRef"
+            :value="formattedValue"
```

Comment:
it can work, but it seems does not link when parent component changed

Reply:
Replaced by v-model